app/evm_keeper.go

```diff
@@ -0,0 +1,85 @@
+package app
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	evmtypes "github.com/tharsis/ethermint/x/evm/types"
+)
+
+// Convertion between native gas KAVA (6) to EVM (18).
+// 12 decimal difference, so 1_000_000_000_000.
+var conversionMultiplier = sdk.NewInt(1_000_000_000_000)
+
+// EVMBankKeeper is a wrapper for bank keeper that converts between EVM and native decimals
+type EVMBankKeeper struct {
+	bankKeeper bankkeeper.Keeper
```

Comment:
Since we intend to only use with the EVM -- we could also depend the abstract https://github.com/tharsis/ethermint/blob/main/x/evm/types/interfaces.go#L29.  Also, it's usually preferable to depend on interfaces where we can (and if it doesn't introduce unnecessary complexity).